engine/npcyclemanager/update.go

```diff
@@ -27,6 +29,41 @@ func (db *DB) update(ctx context.Context, all bool, alertID *int) error {
 	}
 	log.Debugf(ctx, "Updating notification cycles.")
 
-	_, err = db.lock.Exec(ctx, db.queueMessages)
-	return errors.Wrap(err, "queue outgoing messages")
+	tx, err := db.lock.BeginTx(ctx, nil)
+	if err != nil {
+		return err
```

Comment:
let's wrap these since it's in an engine module

messages should just be concise as to what was attempted, e.g. here can be "begin tx"